service/service.go

```diff
@@ -72,17 +79,47 @@ func New(cfg Config, log *mlog.Logger) (*Service, error) {
 
 func (s *Service) Start() error {
 	if err := s.apiServer.Start(); err != nil {
-		return fmt.Errorf("failed to start API server: %w", err)
+		return fmt.Errorf("failed to start api server: %w", err)
+	}
+
+	if err := s.rtcServer.Start(); err != nil {
+		return fmt.Errorf("failed to start rtc server: %w", err)
 	}
 
+	go func() {
+		for msg := range s.wsServer.ReceiveCh() {
+			switch msg.Type {
+			case ws.OpenMessage:
+				s.log.Debug("connect", mlog.String("connID", msg.ConnID))
+			case ws.CloseMessage:
+				s.log.Debug("disconnect", mlog.String("connID", msg.ConnID))
+			case ws.TextMessage:
+				s.log.Warn("unexpected text message", mlog.String("connID", msg.ConnID))
+			case ws.BinaryMessage:
+				var clientMsg ClientMessage
+				if err := clientMsg.Unpack(msg.Data); err != nil {
```

Comment:
Is this going to be a hot path? Should we unpack in a go routine? Or is it negligible?

Reply:
Only time (and data) will tell us. At the moment we have a very low rate of events per second, even when lots of users are joining calls. Keeping it simple for now until further optimization is required.